google/internal/oauth/utils.go

```diff
@@ -0,0 +1,36 @@
+package externalaccount
```

Comment:
Maybe change the name of this file to "clientAuthentication.go", or put all the changes in a file named "utils.go".